src/main/java/org/jvnet/hudson/plugins/repositoryconnector/aether/Aether.java

```diff
@@ -188,38 +185,38 @@ private RepositorySystemSession newSession() {
         return session;
     }
 
-    public AetherResult resolve(String groupId, String artifactId, String classifier, String extension, String version)
-            throws DependencyCollectionException, ArtifactResolutionException, DependencyResolutionException {
+    public AetherResult resolve(String groupId, String artifactId, String classifier, String extension, String version) throws DependencyCollectionException, ArtifactResolutionException, DependencyResolutionException {
         RepositorySystemSession session = newSession();
-        Dependency dependency = new Dependency(new DefaultArtifact(groupId, artifactId, classifier, extension, version), "provided");
-
-        CollectRequest collectRequest = new CollectRequest(dependency, repositories);
-
-        // collectRequest.setRoot(dependency);
-
-        DependencyNode rootNode = repositorySystem.collectDependencies(session, collectRequest).getRoot();
-
-        DependencyRequest dependencyRequest = new DependencyRequest(rootNode, new ExcludeTranisitiveDependencyFilter());
-        DependencyResult resolvedDependencies = repositorySystem.resolveDependencies(session, dependencyRequest);
 
-        PreorderNodeListGenerator nlg = new PreorderNodeListGenerator();
-        rootNode.accept(nlg);
-
-        return new AetherResult(rootNode, nlg.getFiles());
+        ArtifactRequest aReq = new ArtifactRequest();
+        Artifact a = new DefaultArtifact(groupId, artifactId, extension, version);
+        aReq.setArtifact(a);
+        aReq.setRepositories(repositories);
+        
+        ArtifactResult aRes = repositorySystem.resolveArtifact(session, aReq);
+        List<File> files = new ArrayList<File>();
+        files.add(aRes.getArtifact().getFile());
+        
+        AetherResult out = new AetherResult(null, files);
```

Comment:
this changed to NOT include transitive dependencies ... if i want to retrieve a WAR file, for example, i do NOT want to also download all it's dependencies ....